src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -267,7 +267,7 @@ export const WorkspaceDashboard = _.flow(
           h(InfoTile, { title: 'Submissions' }, [submissionsCount]),
           h(InfoTile, { title: 'Access level' }, [roleString[accessLevel]]),
           Utils.canWrite(accessLevel) && h(InfoTile, { title: 'Est. $/month' }, [
-            storageCostEstimate
+            storageCostEstimate || '$ ...'
```

Comment:
I like the look of how this comes up on the page a lot! looks much more clean to me